packages/salesforcedx-vscode-core/package.json

```diff
@@ -579,6 +579,18 @@
           "description": "%telemetry_enabled_description%"
         }
       }
-    }
+    },
+    "jsonValidation": [
+      {
+        "fileMatch": "sfdx-project.json",
+        "url":
+          "https://forcedotcom.github.io/schemas/sfdx-project.json/sfdx-project.schema.json"
```

Comment:
I think we should have these schemas as a module that gets added as a dependency or how would this work in an offline scenario ?

Reply:
We could include them, but even MS uses web versions for most of the built-in schemas. I'm pretty sure they cache them for some period. https://github.com/Microsoft/vscode/blob/master/extensions/javascript/package.json#L101-L129

Reply:
Ok, let's go with this and see if we need to include them locally.